src/modules/WhiteBalance/info.json

```diff
@@ -9,4 +9,4 @@
       }
     },
     "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md"
```

Comment:
Please add the docs links for new modules as well.

Reply:
To all the six new modules?

Reply:
Yes, all 6 of them.